.dockerignore

```diff
@@ -17,3 +18,12 @@ docs
 .yarn-integrity
 Dockerfile
 README.md
+CODE_OF_CONDUCT.md
+CONTRIBUTING.md
+CHANGELOG.md
```

Comment:
I'd keep this in the dockerimage, it has low storage impacts but could be useful to be cat-ted to gather informations regarding the running service.